mayday.go

```diff
@@ -58,8 +68,15 @@ func main() {
 		log.Fatal(err)
 	}
 
+	journals, err := mayday.GetJournals()
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	var tarables []mayday.Tarable
+
 	for i := 0; i < len(files); i++ {
```

Comment:
If we do that, `go` happily reuses `f`, and since we're reusing pointers directly from it to refer to the file handle, it just copies the same file over and over if we use a range loop. I could copy `f` into a new object each time, but that just feels wasteful and unclean...